2022-01-15 13:11:38,274 INFO    MainThread:11232 [wandb_setup.py:_flush():71] setting env: {}
2022-01-15 13:11:38,274 INFO    MainThread:11232 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 13:11:38,274 INFO    MainThread:11232 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-01-15 13:11:38,275 INFO    MainThread:11232 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\nguye\07-Deep-Q-Learning\wandb\run-20220115_131138-gdj71zyn\logs\debug.log
2022-01-15 13:11:38,275 INFO    MainThread:11232 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\nguye\07-Deep-Q-Learning\wandb\run-20220115_131138-gdj71zyn\logs\debug-internal.log
2022-01-15 13:11:38,275 INFO    MainThread:11232 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001C262AE7BB0>
2022-01-15 13:11:38,276 INFO    MainThread:11232 [wandb_init.py:init():404] calling init triggers
2022-01-15 13:11:38,276 INFO    MainThread:11232 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'hyper': 'parameter'}
2022-01-15 13:11:38,276 INFO    MainThread:11232 [wandb_init.py:init():460] starting backend
2022-01-15 13:11:38,276 INFO    MainThread:11232 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-01-15 13:11:38,288 INFO    MainThread:11232 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 13:11:38,550 INFO    MainThread:11232 [backend.py:ensure_launched():221] started backend process with pid: 9976
2022-01-15 13:11:38,551 INFO    MainThread:11232 [wandb_init.py:init():469] backend started and connected
2022-01-15 13:11:38,674 INFO    MainThread:11232 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-15 13:11:38,677 INFO    MainThread:11232 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-15 13:11:38,677 INFO    MainThread:11232 [wandb_init.py:init():533] updated telemetry
2022-01-15 13:11:38,709 INFO    MainThread:11232 [wandb_init.py:init():563] communicating current version
2022-01-15 13:11:40,598 INFO    MainThread:11232 [wandb_init.py:init():568] got version response 
2022-01-15 13:11:40,598 INFO    MainThread:11232 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 13:11:41,053 INFO    MainThread:11232 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 13:11:46,076 INFO    MainThread:11232 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 13:11:46,077 INFO    MainThread:11232 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-15 13:11:46,078 INFO    MainThread:11232 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-15 13:11:46,080 INFO    MainThread:11232 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-15 13:11:46,080 INFO    MainThread:11232 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 13:11:46,085 INFO    MainThread:11232 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-15 13:11:46,085 INFO    MainThread:11232 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-15 13:12:22,834 INFO    MainThread:11232 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-15 13:12:23,057 INFO    MainThread:11232 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-15 13:12:23,057 INFO    MainThread:11232 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-15 13:12:33,647 INFO    MainThread:11232 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-15 13:12:33,649 INFO    MainThread:11232 [wandb_run.py:_restore():1752] restore
2022-01-15 13:12:34,650 INFO    MainThread:11232 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 674
  total_bytes: 674
}

2022-01-15 13:12:35,111 INFO    MainThread:11232 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 674
  total_bytes: 674
}

2022-01-15 13:12:36,122 INFO    MainThread:11232 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 674
  total_bytes: 22524
}

